packit_service/worker/events/pagure.py

```diff
@@ -173,11 +173,15 @@ def get_packages_config(self) -> Optional[PackageConfig]:
     @property
     def repo_url(self) -> Optional[RepoUrl]:
         if not self._repo_url:
-            self._repo_url = RepoUrl.parse(
-                self.packages_config.upstream_project_url
-                if self.packages_config
-                else None
-            )
+            try:
+                upstream_project_url = (
+                    self.packages_config.upstream_project_url
+                    if self.packages_config
+                    else None
+                )
```

Comment:
I am not sure I understand this.
Why should `upstream_project_url` be ambiguous? Because there are multiple packages? In that case shouldn't it happen if we access the data through the `PackageView`, probably I am missing something else?

Reply:
Well, we even have [an issue](https://github.com/packit/packit-service/issues/2172) about it, though that's elsewhere in the code.

Reply:
Perhaps there is a better solution for this problem, but in this particular case `upstream_project_url` is not important (I'm not even sure why it's being accessed, we don't need upstream repo here, perhaps for `propose-downstream` retriggering?).

Reply:
Fixed in https://github.com/packit/packit/pull/2394.